utilities/src/main/java/io/syndesis/qe/bdd/CommonSteps.java

```diff
@@ -159,4 +167,27 @@ public void deployMySQLServer() {
     public void createStandardDBSchemaOn(String dbType) {
         new DbUtils(dbType).createSEmptyTableSchema();
     }
+
+    @Given("^allocate new \"([^\"]*)\" database for \"([^\"]*)\" connection$")
+    public void allocateNewDatabase(String dbLabel, String connectionName) {
+
+        dbAllocatorClient.allocate(dbLabel);
+        log.info("Allocated database: '{}'", dbAllocatorClient.getDbAllocation());
+        TestUtils.setDatabaseCredentials(connectionName.toLowerCase(), dbAllocatorClient.getDbAllocation());
+    }
+
+    @Given("^frees allocated \"([^\"]*)\" database$")
+    public void freesAllocatedDatabase(String dbLabel) {
```

Comment:
Probably just typo, please just `freeAllocatedDatabase`, lets not let gherkin steps into java names ;-)